src/Python-2.0.1.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license licenseId="Python-2.0.1" name="Python Software
+   Foundation License 2.0.1" listVersionAdded="3.18">
```

Comment:
Should the license name be "Python License 2.0.1" rather than "Python Software Foundation License 2.0.1"? I think we've used the latter name for the `PSF` version, not for the full `Python` stack.

Reply:
good point, fixed